source/includes/steps-install-shell-linux-rpm.yaml

```diff
@@ -50,7 +50,7 @@ content: |
 
                [mongodb-org-{+mdb-version+}]
                name=MongoDB Repository
-               baseurl=https://repo.mongodb.org/yum/amazon/$releasever/mongodb-org/{+mdb-version+}/$basearch/
+               baseurl=https://repo.mongodb.org/yum/amazon/2023/mongodb-org/{+mdb-version+}/$basearch/
                gpgcheck=1
                enabled=1
                gpgkey=https://www.mongodb.org/static/pgp/server-{+pgp-version+}.asc 
```

Comment:
[suggestion] I can't comment directly on the line, but on line 63 it might help to add `2023` as an example for Amazon Linux version since that's what's used in the example above:

> Amazon Linux version (for example, ``2023``)

Reply:
Done!